### ISSUE TITLE
PostGIS functions are case sensitive

### ISSUE THREAD

**Issue by adrien-berchet** [geoalchemy/geoalchemy2#215]
Hi,
We just discovered that the PostGIS functions are case sensitive when called with the following syntax:
```python
from sqlalchemy import select, func
conn.execute(select([func.ST_Transform(table.c.geometry, 2154)])).fetchall()
conn.execute(select([func.st_transform(table.c.geometry, 2154)])).fetchall()
```
The first one returns a `WKBElement` as expected while the second one returns the EWKB as string. On the contrary, `select([city_objects.c.geometry.ST_Transform(2154)])` returns the expected `WKBElement` while `select([city_objects.c.geometry.st_transform(2154)])` raises the following error: `AttributeError: Neither 'Column' object nor 'Comparator' object has an attribute 'st_transform'`.

I think that these behaviours are quite confusing and could be improved. I see 2 solutions: either raising the same Attribute error when calling `func.st_transform` (or any lowercase function), or make function calls case insensitive.

I think I prefer the second option. What do you think about it?

**Comment by elemoine** [comment]
Excellent work on https://github.com/sqlalchemy/sqlalchemy/commit/702c96b2ee94a394ab1b4d528e3090c4d3c42b91 @adrien-berchet! Does this mean there's nothing to do on the GeoA side?

**Comment by adrien-berchet** [comment]
Thanks :blush: 
I will do some tests but I think that sqlalchemy/sqlalchemy@702c96b fixes almost everything for this issue.

**Comment by adrien-berchet** [comment]
Fixed in #216
